ivy/functional/ivy/linear_algebra.py

```diff
@@ -758,13 +758,16 @@ def eigvalsh(
 
     >>> x = ivy.array([[[3.0,2.0],[2.0,3.0]]])
     >>> y = ivy.zeros([1,2])
-    >>> ivy.eigvalsh(x, out=y)
-    >>> print(y)
+    >>> ivy.eigvalsh(x)
```

Comment:
If you do not pass the y in parameter, how will the output be written to y ?

Reply:
Ohh, missed that, will make the changes right away